api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/dynamodb/AbstractDynamoDbProperties.java

```diff
@@ -45,6 +45,22 @@
      */
     private long writeCapacity = 10;
 
+    /**
+     * Billing mode specifies how you are charged for read and write throughput
+     * and how you manage capacity.
+     *
+     * Accepted values are {@code PROVISIONED, PAY_PER_REQUEST}.
```

Comment:
No need to specify accepted values; They will be picked up automatically from the configuration metadata.